tools/hardware/outputs/c_header.py

```diff
@@ -36,6 +36,14 @@
 #include <linker.h>         /* for BOOT_RODATA */
 #include <basic_types.h>    /* for p_region_t, kernel_frame_t (arch/types.h) */
 
+/* wrap raw physBase location constant to give it a symbolic name in C that's
+ * visible to verification
+ */
+static inline word_t physBase(void)
```

Comment:
We could state in the interface explicitly that this is supposed to return a constant.
```suggestion
static inline CONST word_t physBase(void)
```
The compiler should be smart enough to figure this out from the code in the body, so I'm not sure if there is much difference. However, it allows the compiler to raise some warning if for whatever reason the code in the body does not produce a constant.

Reply:
adding CONST makes sense, yes

Reply:
- [ ] add CONST